ethereum/spec/src/property-test/java/tech/pegasys/teku/spec/datastructures/execution/SignedBuilderBidPropertyTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.datastructures.execution;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import net.jqwik.api.Arbitraries;
+import net.jqwik.api.Arbitrary;
+import net.jqwik.api.ForAll;
+import net.jqwik.api.Property;
+import net.jqwik.api.Provide;
+import tech.pegasys.teku.infrastructure.json.JsonUtil;
+import tech.pegasys.teku.infrastructure.json.types.DeserializableTypeDefinition;
+import tech.pegasys.teku.spec.Spec;
+import tech.pegasys.teku.spec.SpecMilestone;
+import tech.pegasys.teku.spec.TestSpecFactory;
+import tech.pegasys.teku.spec.networks.Eth2Network;
+import tech.pegasys.teku.spec.util.DataStructureUtil;
+
+public class SignedBuilderBidPropertyTest {
+  @Property
+  void roundTrip(
+      @ForAll final int seed,
+      @ForAll("milestone") final SpecMilestone specMilestone,
+      @ForAll final Eth2Network network)
+      throws JsonProcessingException {
+    final Spec spec = TestSpecFactory.create(specMilestone, network);
+    final DataStructureUtil dataStructureUtil = new DataStructureUtil(seed, spec);
+    final SignedBuilderBid bid = dataStructureUtil.randomSignedBuilderBid();
+
+    final ExecutionPayloadHeader header = dataStructureUtil.randomExecutionPayloadHeader();
+    final BuilderBidSchema schema = new BuilderBidSchema(header.getSchema());
+    final SignedBuilderBidSchema signedSchema = new SignedBuilderBidSchema(schema);
```

Comment:
For the record, these schemas from the builder API are a bit unique - they are the first ones we have that aren't from the beacon chain spec and so versioning won't be tied to hard forks. We don't have a central repository for them currently, so we wind up creating new instances like this.  Might be something we want to revisit though as creating SSZ schemas can be expensive so we probably want to make sure we're reusing them in production.